core/lib/dal/src/blocks_web3_dal.rs

```diff
@@ -219,7 +223,7 @@ impl BlocksWeb3Dal<'_, '_> {
                         .into(),
                     base_fee_per_gas: Some(bigdecimal_to_u256(row.base_fee_per_gas.clone())),
                     extra_data: Bytes::default(),
-                    logs_bloom: H2048::default(),
+                    logs_bloom: Bloom::default(),
```

Comment:
Dumb question: Why is the default value OK here? AFAIU, this method is used in subscriptions, and the `logsBloom` field is visible there.

Reply:
This place was missed, nice catch!